docs/source/examples/performance_modeling_output.txt

```diff
@@ -17,44 +17,24 @@ W_{burn} : [ 0.487     0.486     0.485     0.485    ]  [lbf] segment fuel burn
 W_{fuel} : [ 1.94      1.46      0.97      0.485    ]  [lbf] fuel weight
 
          | Mission/FlightSegment/AircraftP/WingAero
-     C_D : [ 0.00783   0.00782   0.00781   0.0078   ]        drag coefficient
-     C_L : [ 0.47      0.469     0.468     0.467    ]        lift coefficient
        D : [ 4.87      4.86      4.85      4.85     ]  [lbf] drag force
-      Re : [ 7.56e+05  7.56e+05  7.56e+05  7.56e+05 ]        Reynold's number
 
-Constants
----------
+Sensitivities
+-------------
      | Aircraft/Fuselage
-   W : 100                                         [lbf]       weight
+   W : +0.25            weight
```

Comment:
I tried suggesting that, everyone kept shooting it down. And then I tried it and didn't like it myself!